packages/pages-components/src/components/analytics/interfaces.ts

```diff
@@ -95,6 +95,11 @@ export interface AnalyticsProviderProps {
    * and will NOT be fired/sent through the Analytics SDK.
    */
   enableDebugging?: boolean | undefined;
+
+  /**
+   * productionDomains marks that these domains are in production and analytics should run.
```

Comment:
productionDomains is an optional list of domains that analytics should fire on. If your production site is hosted by Yext then this field is not required. It is only necessary if this code is being run in the context of a Yext module.